model/UserModel.ts

```diff
@@ -1,5 +1,153 @@
+import type { UidIdentifier } from "firebase-admin/auth";
+
 export class UserModel {
-    /**
-     * Model for the user
-     */
-}
\ No newline at end of file
+    //private fields for the user's statistics
+
+    //name of the user
+    private _userName: string = '';
+
+    //current streak of the user
+    private _currentStreak: number = 0;
+    //max streak of the user
+    private _maxStreak: number = 0;
+
```

Comment:
For the _currentStreak you should also be able to retrieve the last day the player has played, if it was yesterday then you increase the currentStreak else you set it to 0 (or 1 if they just played).
Maybe you planned on doing it another way but I don't see it in you code. Like on what condition do you call resetStreak ?

Reply:
Planned with issue #29.